orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/OrgDisambiguatedDaoImpl.java

```diff
@@ -87,6 +85,19 @@ public OrgDisambiguatedEntity findByNameCityRegionCountryAndSourceType(String na
         return query.getResultList();
     }
 
+    @Override
+    public List<OrgDisambiguatedEntity> findOrgsToGroup(int firstResult, int maxResult) {
+        TypedQuery<OrgDisambiguatedEntity> query = entityManager.createQuery("from OrgDisambiguatedEntity where source_type=:ROR or source_type=:FUNDREF or source_type=:RINGGOLD",
+                OrgDisambiguatedEntity.class);
+        //group by ROR
```

Comment:
Why GRID orgs are not included in this query?

Reply:
@angel  as per card https://trello.com/c/8kwEskUt/7825-include-ror-orgs-in-typeahead  and part of cleaning GRID I removed the GRID from the query as other organisations types that are "retired". If it was a misunderstanding I can easily add it back.